packages/create-agentica/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "create-agentica",
+  "version": "0.1.0",
+  "description": "alias of agentica cli",
+  "type": "module",
+  "bin": "./index.js",
+  "author": "Wrtn Technologies",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wrtnlabs/agentica"
```

Comment:
this file should have "scripts.build" property

Reply:
I don't think we need any build step for now

Reply:
I don't think we need any build step for now

Reply:
just for release step! (https://github.com/wrtnlabs/agentica/actions/runs/13961535019/job/39083624002)

already added!